apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -115,4 +134,25 @@ export class VersionControl {
       .toString();
     return output.trim().split('\n');
   }
+
+  private static _tryFetchRemoteBranch(remoteBranchName: string): boolean {
+    const firstSlashIndex: number = remoteBranchName.indexOf('/');
```

Comment:
Since we're about to perform a network action that modifies the state of the working folder, perhaps we should log an informational message here.

I'm thinking something like: `Checking for updates to origin/master...`